packages/drivers/routerlicious-driver/src/shreddedSummaryDocumentStorageService.ts

```diff
@@ -165,13 +165,8 @@ export class ShreddedSummaryDocumentStorageService implements IDocumentStorageSe
     }
 
     public async uploadSummaryWithContext(summary: ISummaryTree, context: ISummaryContext): Promise<string> {
-        const summaryHandle = await PerformanceEvent.timedExecAsync(
-            this.logger,
-            {
-                eventName: "uploadSummaryWithContext",
-            },
-            async () => this.summaryUploadManager.writeSummaryTree(summary, context.ackHandle ?? "", "channel"),
-        );
+        const summaryHandle = await this.summaryUploadManager.writeSummaryTree(
+            summary, context.ackHandle ?? "", "channel");
```

Comment:
I don't think we should remove this event from the routerlicious-driver since it doesn't have the `uploadSummary` event.

Reply:
Or maybe it does have `uploadSummary` event and I missed it because its not modified in this PR :-).